reader.py

```
@@ -301,7 +301,7 @@ def save_progress(self):
         with RepoWorker("D://adams-comics//.covers") as page_saver:
             if page == 0:
                 return None
```

Ruff (0.12.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)